docs/index.rst

```diff
@@ -0,0 +1,12 @@
+.. _home:
+
+Apache Kafka
+------------
+
+.. toctree::
+    :maxdepth: 1
+
+    introduction
+    quickstart
+    uses
+    documentation/index
```

Comment:
@ewencp agreed. here's my idea: remove the 2nd `Introduction`, since it is a duplicate. and then won't the TOC actually look like:
HOME
INTRODUCTION
QUICKSTART
USE CASES
DOCUMENTATION
- Kafka APIs
- Configuration
- Design
- Implementation
- Operations
- Security
- Kafka Connect
- Kafka Streams
PERFORMANCE
POWERED BY
PROJECT INFO
ECOSYSTEM
CLIENTS
EVENTS
CONTACT US
APACHE
Download
@apachekafka
PUBLISH

Reply:
Currently the `Intro` is just a quick link into the documentation section. I'm +1 for removing the second introduction link.